src/components/pill/pill.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import Pill from './pill';
+import notes from './notes.md';
+import OptionsHelper from '../../utils/helpers/options-helper';
+
+storiesOf('Pill', module)
+  .add('default', () => {
+    const children = text('children', 'Pill');
+    const as = children ? select('as', OptionsHelper.colors, Pill.defaultProps.as) : undefined;
+    const fill = children ? boolean('fill', Pill.defaultProps.fill) : undefined;
+    const onDelete = boolean('onDelete', false);
+
+    return (
```

Comment:
Again the component is pretty much overwhelmed by the title underneath it.

![image](https://user-images.githubusercontent.com/26925206/53879710-08e9ce80-4007-11e9-8f4c-e2a2a8d170c2.png)

Reply:
Fixed as per my other comment 👍